pybossa/core.py

```diff
@@ -666,7 +666,8 @@ def get_wizard_steps(project=None):
             notify_admin=notify_admin,
             plugins=plugins,
             ldap_enabled=ldap_enabled,
-            get_wizard_steps=get_wizard_steps)
+            get_wizard_steps=get_wizard_steps,
+            server_type=app.config['SERVER_TYPE'])
```

Comment:
If `SERVER_TYPE` is optional and can be an empty string, could we default this value to prevent an exception when loading? :)

```python
server_type=app.config.get('SERVER_TYPE', '')
```